wiki/BanchoBot/de.md

```diff
@@ -1,25 +1,158 @@
 ---
-outdated: true
+tags:
+  - bancho
+  - server bot
+  - commands
+  - announcements
+  - Befehle
+  - Ankündigungen
 ---
 
 # BanchoBot
 
-![BanchoBots Benutzerpanel.](img/BanchoBot.jpg "BanchoBots Benutzerpanel.")
+*Für andere Verwendungen, siehe [Bancho (Begriffsabgrenzung)](/wiki/Disambiguation/Bancho).*
 
-BanchoBot ist ein Bot (d.h. er führt eine Liste an Befehlen aus und antwortet mit einem Ergebnis), der von **Echo49** entwickelt wurde. Er hilft Spielern im Chat, indem er nützliche Informationen anzeigt und wichtige spielbezogene Nachrichten verkündet. Er ist der [Host](https://de.wikipedia.org/wiki/Hostrechner) des [bancho irc](/wiki/Community/Internet_Relay_Chat) und hat ein Nutzerprofil mit der ID [3](https://osu.ppy.sh/users/3) (neben peppy mit der ID [2](https://osu.ppy.sh/users/2)). *Außerdem hat er ein eigenes [Twitterprofil](https://twitter.com/banchoboat)!*
+![BanchoBot's Benutzerkarte](img/BanchoBot.jpg "BanchoBot's Benutzerkarte")
 
-## BanchoBots Befehle
+**BanchoBot** (manchmal auch *Bancho* genannt) ist ein Online-Chat-Bot, der für osu! entwickelt wurde und den Spielern im Chat hilft, indem er spielbezogene Nachrichten (z. B. Anzahl der Plays, wiederholte Versuche, usw.) ansagt und auf bestimmte Befehle reagiert. Es wurde von [Echo](https://osu.ppy.sh/users/431) programmiert und ist der Bereitsteller des [Bancho IRC](/wiki/Community/Internet_Relay_Chat) (Internet Relay Chat).
```

Comment:
```suggestion
**BanchoBot** (manchmal auch *Bancho* genannt) ist ein Online-Chat-Bot, der für osu! entwickelt wurde und den Spielern im Chat hilft, indem er spielbezogene Nachrichten (z. B. Anzahl der Plays, wiederholte Versuche, usw.) ansagt und auf bestimmte Befehle reagiert. Er wurde von [Echo](https://osu.ppy.sh/users/431) programmiert und ist der Bereitsteller des [Bancho IRCs](/wiki/Community/Internet_Relay_Chat) (Internet Relay Chat).
```
1. [Duden suggests `der Bot`](https://www.duden.de/rechtschreibung/Bot)
2. [`des Chats`](https://www.duden.de/deklination/substantive/Chat); `stellt den Bancho IRC bereit` might be possible too, in case you want to go around the genitiv form